generators/client-build/templates/gulp/tasks/watch-client.js.tpl

```diff
@@ -20,10 +21,13 @@ gulp.task('watch-client', () => {
     gulp.watch(config.client.staticJade, ['process-jade']);<% } %>
 
     // watch sass
-    gulp.watch(config.client.styles, ['sass']);<% if (client === 'react') { %>
-    // watch client js
-    lrload.monitor(config.client.dist.path + '/' + config.client.dist.bundle, {displayNotification: true});<% } %>
+    lload.listen();
+    gulp.watch(config.client.styles, ['sass']);
 
+<% if (client === 'react') { %>
```

Comment:
Yeah I just had to make sure there were no spaces before expression blocks

```
<% if (client === 'react') { %>
    derp
<% } %>
```

vs 
```
    <% if (client === 'react') { %>
    derp
    <% } %>
```